src/Raven.Studio/typescript/viewmodels/database/tasks/exportDatabase.ts

```diff
@@ -239,6 +231,42 @@ class exportDatabase extends viewModelBase {
                     }).always(() => exportDatabase.isExporting(false));
             });
     }
+
+    getCommandTypeLabel(cmdType: commandLineType) {
+        return `Export Command - ${cmdType}`;
+    }
+
+    copyCommandToClipboard() {
+        const command = this.effectiveCommand();
+        copyToClipboard.copy(command, "Export command was copied to clipboard.");
+    }
+    
+    getCommand(commandType: commandLineType) {
```

Comment:
getCommand() should return [string, string]. First will be file name, second json. You don't need commandType arg here.